doc/release/contribs.py

```diff
@@ -0,0 +1,203 @@
+#!/usr/bin/env python
```

Comment:
Did this file get added to the PR by accident? It does not seem related to anything else here.